docs/develop/tutorial/2-building/index.mdx

```diff
@@ -238,7 +238,7 @@ If working with older example code that references `COMPACT_HOME`, you have two
 2. **Set COMPACT_HOME for compatibility**: Export the variable pointing to `~/.compact/bin` for temporary backward compatibility
 
 **Direct compiler access**: While not recommended, the installed toolchain binaries reside in `~/.compact/bin/`. This directory contains symbolic links to the current default version's binaries:
-- `compactc` - The main compiler executable
+- `compact` - The main compiler executable
 - `zkir` - Zero-knowledge intermediate representation tool
```

Comment:
And I don't think it puts `zkir` in there (see above) because it's not a command entry point.

This page needs a bigger rewrite for accuracy.